packages/@react-spectrum/card/stories/Card.stories.tsx

```diff
@@ -10,186 +10,62 @@
+      </>
+    )
+  },
+  decorators: [
+    (Story) => (
+      <div style={{width: '208px'}}>
+        <Story />
+      </div>
+    )
+  ]
 };
 
-export default meta;
-
-
-const Template = (): Story<SpectrumCardProps> => (args) => (
-  <div style={{width: '208px'}}>
-    <Card {...args} />
-  </div>
-);
-
-/* This is a bit of a funny template, we can't get selected on a Card through context because
-* if there's context it assumes it's being rendered in a collection. It's just here for a quick check of styles. */
-interface ISelectableCard {
-  disabledKeys: Set<any>,
-  selectionManager: {
-    isSelected: () => boolean,
-    select: () => Dispatch<SetStateAction<ISelectableCard>>
-  }
-}
-let SelectableCard = (props) => {
-  let [state, setState] = useState<ISelectableCard>({
-    disabledKeys: new Set(),
-    selectionManager: {
-      isSelected: () => true,
-      select: () => setState(prev => ({
-        ...prev,
-        selectionManager: {
-          ...prev.selectionManager,
-          isSelected: () => !prev.selectionManager.isSelected()
-        }
-      }))
-    }
-  });
-  let {pressProps} = usePress({onPress: () => setState(prev => ({
-    ...prev,
-    selectionManager: {
-      ...prev.selectionManager,
-      isSelected: () => !prev.selectionManager.isSelected()
-    }
-  }))});
-  return (
-    <div style={{width: '208px'}} {...pressProps}>
-      <CardViewContext.Provider value={{state}}>
-        <CardBase {...props} />
-      </CardViewContext.Provider>
-    </div>
-  );
+export const CardGrid: CardStory = {
+  render: (args, context) => <Card {...args} {...context} />,
+  args: {layout: 'grid'},
+  decorators: [(Story, context) => (
+    <CardGridDecorator
+      Story={Story}
+      context={context} />
+  )]
 };
-const TemplateSelected = (): Story<SpectrumCardProps> => (args) => (
-  <div style={{width: '208px'}}>
-    <SelectableCard {...args} />
-  </div>
-);
-
-
-export const Default = Template().bind({});
-Default.args = {children: (
-  <>
-    <Image src="https://i.imgur.com/Z7AzH2c.jpg" />
-    <Avatar src="https://mir-s3-cdn-cf.behance.net/project_modules/disp/690bc6105945313.5f84bfc9de488.png" />
-    <Heading>Title</Heading>
-    <Text slot="detail">PNG</Text>
-    <Content>Description</Content>
-  </>
-)};
-
-export const DefaultSquare = Template().bind({});
-DefaultSquare.args = {children: (
-  <>
-    <Image src="https://i.imgur.com/DhygPot.jpg" />
-    <Heading>Title</Heading>
-    <Text slot="detail">PNG</Text>
-    <Content>Description</Content>
-  </>
-)};
-
-export const DefaultTall = Template().bind({});
-DefaultTall.args = {children: (
-  <>
-    <Image src="https://i.imgur.com/3lzeoK7.jpg" />
-    <Heading>Title</Heading>
-    <Text slot="detail">PNG</Text>
-    <Content>Description</Content>
-  </>
-)};
-
-export const DefaultPreviewAlt = Template().bind({});
-DefaultPreviewAlt.args = {children: (
-  <>
-    <Image alt="preview" src="https://i.imgur.com/Z7AzH2c.jpg" />
```

Comment:
specifically left off of some stories because if an alt isn't provided, it's decorative, chromatic will eventually check the accessibility tab, so we'll be alerted if we break it there